src/components/HeaderForm.js

```diff
@@ -825,6 +832,19 @@ class HeaderForm extends Component {
     this.setState({ simplify: !this.state.simplify });
   };
 
+  /* Function for toggling nodeSequences button, enabling client to make request for node sequences to be displayed or note */
```

Comment:
I don't think `nodeSequences` is a variable or function name, and `note` is a typo. We could simplify the sentence to something like:

```suggestion
  /* Function for toggling display of node sequences */
```